libres/lib/include/ert/analysis/ies/ies_config.hpp

```diff
@@ -0,0 +1,72 @@
+/*
+   Copyright (C) 2019  Equinor ASA, Norway.
+
+   The file 'ies_enkf_config.hpp' is part of ERT - Ensemble based Reservoir Tool.
+
+   ERT is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or
+   FITNESS FOR A PARTICULAR PURPOSE.
+
+   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+   for more details.
+*/
+
+#ifndef IES_CONFIG_H
+#define IES_CONFIG_H
+
+#include <stdbool.h>
```

Comment:
This includes does nothing in C++, you can remove it.

Reply:
Done